------------------------------------------------------------------------------------------------------------------------

[2018-10-25 15:18:48 #1 server] 
Server on port 6136 has started by 9852@ChiChaChai

------------------------------------------------------------------------------------------------------------------------

[2018-10-25 15:21:13 #1 server] 
java.net.BindException: Address already in use: JVM_Bind
	at java.net.DualStackPlainSocketImpl.bind0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketBind(Unknown Source)
	at java.net.AbstractPlainSocketImpl.bind(Unknown Source)
	at java.net.PlainSocketImpl.bind(Unknown Source)
	at java.net.ServerSocket.bind(Unknown Source)
	at java.net.ServerSocket.<init>(Unknown Source)
	at java.net.ServerSocket.<init>(Unknown Source)
	at com.objectdb.o.SMR.o(SMR.java:260)
	at com.objectdb.o.SMR.f(SMR.java:160)
	at com.objectdb.o.TOL.run(TOL.java:117)
	at com.objectdb.Server.runCommand(Server.java:193)
	at com.objectdb.Server.run(Server.java:104)
	at com.objectdb.Server.main(Server.java:63)

------------------------------------------------------------------------------------------------------------------------

[2018-10-25 15:21:27 #1 store.file] 
readTextFile error.

[2018-10-25 15:21:27 #2 store.file] 
java.io.FileNotFoundException: C:\Users\ChiChaChai\.objectdb\exolorer.xml (The system cannot find the file specified)
	at java.io.FileInputStream.open0(Native Method)
	at java.io.FileInputStream.open(Unknown Source)
	at java.io.FileInputStream.<init>(Unknown Source)
	at java.io.FileInputStream.<init>(Unknown Source)
	at com.objectdb.o.STH.A(STH.java:717)
	at com.objectdb.o.CNF.<init>(CNF.java:106)
	at com.objectdb.o.CNF.<clinit>(CNF.java:32)
	at com.objectdb.o.MainFrame.init(MainFrame.java:132)
	at com.objectdb.o.EXA.<init>(EXA.java:77)
	at com.objectdb.Explorer$1.run(Explorer.java:57)
	at java.awt.event.InvocationEvent.dispatch(Unknown Source)
	at java.awt.EventQueue.dispatchEventImpl(Unknown Source)
	at java.awt.EventQueue.access$500(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.awt.EventQueue.dispatchEvent(Unknown Source)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.run(Unknown Source)

[2018-10-25 15:22:09 #3 store] 
Database 'C:\Users\ChiChaChai\Desktop\i-classroom\objectdb\db\points.odb' is opened by 22304@ChiChaChai

[2018-10-25 15:22:09 #4 store] 
Configuration path: C:\Users\ChiChaChai\Desktop\i-classroom\objectdb\objectdb.conf

[2018-10-25 15:28:05 #5 store] 
Database 'C:\Users\ChiChaChai\Desktop\i-classroom\objectdb\db\points.odb' is closed by 22304@ChiChaChai

